index.html

```diff
@@ -40,41 +42,59 @@
           <i class="fa fa-hand-o-down fa-lg"></i>
         </footer>
       </section>
-
-      <section class="section" id="page2">
-        <section class="placeholder bottom right">
-          <header>Fish Fillet</header>
-          <section>Fish fillets comprise the flesh of the fish, which is the skeletal muscles and fat as opposed to the bones and viscera.</section>
-          <footer>
-            <i class="fa fa-usd"></i>
-            <span>5.50</span>
-          </footer>
-        </section>
-      </section>
-
-      <section class="section" id="page3">
-        <section class="placeholder top left">
-          <header>Pizza</header>
-          <section>Oven-baked flat bread generally topped with tomato sauce and cheese. It is commonly supplemented with a selection of meats, vegetables and condiments.</section>
-          <footer>
-            <i class="fa fa-usd"></i>
-            <span>6.45</span>
-          </footer>
-        </section>
-      </section>
-
-      <section class="section" id="page4">
-        <section class="placeholder top left">
-          <header>Beef Steak</header>
-          <section>A cut of meat sliced perpendicular to the muscle fibers, potentially including a bone.</section>
-          <footer>
-            <i class="fa fa-usd"></i>
-            <span>6.45</span>
-          </footer>
-        </section>
-      </section>
+      
 
     </section>
-
+    
+    <!-- JQuery -->
+    <script>
+      //JSON object
+      var x = [ 
+                {
+                  "name":"Fish Fillet",
+                  "description":"Fish fillets comprise the flesh of the fish, which is the skeletal muscles and fat as opposed to the bones and viscera.",
+                  "price":"5.50",
+                  "placeholder":"bottom right",
+                  "url":"img/menu/fish_bg.jpg"
+                },
+                {
+                  "name":"Pizza",
+                  "description":"Oven-baked flat bread generally topped with tomato sauce and cheese. It is commonly supplemented with a selection of meats, vegetables and condiments.",
+                  "price":"6.45",
+                  "placeholder":"top left",
+                  "url":"img/menu/pizza_bg.jpg"
+                },
+                {
+                  "name":"Beef Steak",
+                  "description":"A cut of meat sliced perpendicular to the muscle fibers, potentially including a bone.",
+                  "price":"6.45",
+                  "placeholder":"top left",
+                  "url":"img/menu/steak_bg.jpeg"
+                }
+              ];
+      var pageElement = '<section class="section page">' +
+                            '<section>' +
```

Comment:
I don't think you need to add class ```placeholder``` dynamically below. you can just put it here